simulation/__main__.py

```diff
@@ -148,6 +148,12 @@ def check_string_is_isoformat_datetime(string) -> str:
         type=convert_to_datetime,
     )
 
+    parser.add_argument(
+        "--gas-oil-boiler-ban-announce-date",
+        default=datetime.datetime(2025, 1, 1),
+        type=convert_to_datetime,
```

Comment:
I think we also need a check in here that `gas_oil_boiler_ban_announce_date < gas_oil_boiler_ban_date`, perhaps via an assert statement like this somewhere:
```
assert gas_oil_boiler_ban_announce_date < gas_oil_boiler_ban_date, "Ensure the gas oil boiler ban announcement date happens after the ban date"
```

Reply:
Also when we do this, add a test for this behaviour